logstash-input-sqs.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency "logstash-core", '>= 1.4.0', '< 2.0.0'
 
   s.add_runtime_dependency 'logstash-codec-json'
-  s.add_runtime_dependency 'aws-sdk'
+  s.add_runtime_dependency 'aws-sdk', '~> 2.1.0'
```

Comment:
We don't need to depend directly on aws-sdk v2.
We can just define a dependencie on logstash-mixin-aws, like this:

```ruby
  s.add_runtime_dependency "logstash-mixin-aws", ">= 1.0.0"
```